src/main/fc/rc.c

```diff
@@ -967,6 +966,22 @@ void initRcProcessing(void)
     const int maxYawRate = (int)maxRcRate[FD_YAW];
     initYawSpinRecovery(maxYawRate);
 #endif
+
+    if (rxConfig()->rc_smoothing_use_tau) {
+        const float cen_tau = currentControlRateProfile->rc_smoothing_setpoint_tau_center;
+        const float end_tau = currentControlRateProfile->rc_smoothing_setpoint_tau_end;
+        const float throttle_tau = currentControlRateProfile->rc_smoothing_throttle_tau;
+        // unit is 10th of a millisecond, sort of
+        rcSmoothingData.setpointTauCenter = cen_tau / 10000.0f + cen_tau * cen_tau / 1250000.0f;
```

Comment:
Maybe use 16bit tau ... This will work great until you try to explain it to anyone ;-)

Reply:
I mean I could, you then just need a range of like 0-2000+ and do hundreths of a ms as your unit. If you want like a 100hz cutoff or higher you'd need that much precision and this high of value to still have lots of smoothing. This should be easier to understand than the current rc smoothing value however. But we could just move this to a 16bit.

Reply:
I'll probably just end up testing both on my end and see what ends up feeling right.

Reply:
I'd like to implement float16 in CLI. That would help with tons of strange scaling.

Reply:
Float16 could be interesting.